_docs/templates/_npe2_menus_guide.md.jinja

```diff
@@ -0,0 +1,91 @@
+(menus-contribution-guide)=
+## Menus
+
+Menu contributions enable plugin developers to add new items or submenus to *existing* napari menus, allowing commands to be executed directly through menu interface. This is a great mechanism for exposing simple commands that might be overkill for a widget.
```

Comment:
Just want to tweak this as the menu contributions will often still be widgets. I think the principle is that even if they are widgets, they should apply to a wide range of data, rather than a niche use-case served by the plugin.

```suggestion
Menu contributions enable plugin developers to add new items or submenus to *existing* napari menus, allowing commands to be executed directly through the menu interface. This is a great mechanism for exposing simple commands or widgets that are generally applicable to a wide range of data.
```